### ISSUE TITLE
Room Preview buttons have a contrast ratio of 2.4:1 when it should be 3:1

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#14731]
Green buttons are bad with white text

**Comment by erikkroes** [comment]
Everywhere where mint meets white or grey, it's going to be a challenge. Implementing aXe like in #14732 might find a lot of these. If there's a style guide or something alike, it's worth to double check it for usable combinations.

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/riot-web/issues/14501

**Comment by t3chguy** [comment]
Re-opening as #14501 was re-scoped to just be for the heading for release urgency

**Comment by germain-gg** [comment]
Fixed by https://github.com/matrix-org/matrix-react-sdk/pull/11079 we're now using a different shade a green that meets accessibility requirements
